src/tools/wasm-split/instrumenter.cpp

```diff
@@ -28,19 +28,22 @@ Instrumenter::Instrumenter(const WasmSplitOptions& options, uint64_t moduleHash)
 void Instrumenter::run(PassRunner* runner, Module* wasm) {
   this->runner = runner;
   this->wasm = wasm;
-  addGlobals();
+  size_t numFuncs = 0;
+  ModuleUtils::iterDefinedFunctions(*wasm, [&](Function*) { ++numFuncs; });
```

Comment:
It would be nice to give this some space to hint visually that the two lines work together.
```suggestion

  size_t numFuncs = 0;
  ModuleUtils::iterDefinedFunctions(*wasm, [&](Function*) { ++numFuncs; });
  
```